pkgs/development/libraries/libnfc/default.nix

```diff
@@ -1,15 +1,25 @@
-{ stdenv, fetchurl, libusb, readline }:
+{ stdenv, fetchFromGitHub
+, cmake, pkgconfig
+, libusb, pcsclite, acsccid }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "libnfc";
-  version = "1.7.1";
+  version = "2019-08-21";
```

Comment:
why do we need an unstable version for this? also the version should be "unstable-2019-08-21" then

Reply:
Late response like no other, but it for building `ykchalresp-nfc`. Don't know if this before or after I committed but I thought the convention changed.